community/server/src/main/java/org/neo4j/server/rest/web/DatabaseActions.java

```diff
@@ -17,6 +17,7 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
+
 package org.neo4j.server.rest.web;
```

Comment:
There are very few changes in this file, the rest is just code style. It would give a better overview to not format the entire file when changing a couple of lines.